addons/sale/report/sale_report.py

```diff
@@ -33,7 +33,7 @@ def _get_done_states(self):
             ('invoiced', "Fully Invoiced"),
             ('to invoice', "To Invoice"),
             ('no', "Nothing to Invoice"),
-        ], string="Invoice Status", readonly=True)
+        ], string="Order Invoice Status", readonly=True)
```

Comment:
I think you could just use this field instead of adding new one and update query accordingly